### PR TITLE
add alicloud bat spec

### PR DESCRIPTION
**Pull request by demonwy**
We have run pass the BATs specs, now submit the code.
Node: Alicloud disk size minimum value is 20GB, so we change the with_release_stemcell_spec disk size to 20G to pass through.

refer for Alicloud Disk API: https://www.alibabacloud.com/help/doc-detail/25513.htm?spm=a3c0i.o25429en.b99.193.6ad6eb95iQhGf7

### CODE REVIEW

File: shared/tasks/run-bats.sh [hunk 1]
```diff
@@ -31,6 +31,12 @@ chmod 600 $ssh_key_path
 export BOSH_GW_PRIVATE_KEY=$ssh_key_path
 
 pushd bats
+  # mock: alicloud does not support disk size less than 20GB
+  if [ ${INFRASTRUCTURE} == "alicloud" ];
+  then
+    sed -ig "s/2048/20480/" ./spec/system/with_release_stemcell_spec.rb
+    sed -ig "s/4096/40960/" ./spec/system/with_release_stemcell_spec.rb
```
Comment (dpb587-pivotal) [inline shared/tasks/run-bats.sh]: @demonwy this is fragile. Does the CPI automatically upsize disks which are smaller than the IaaS minimum?

@cdutra would it be acceptable/normal for a CPI to automatically give a larger disk if the IaaS doesn't support the smaller, user-requested size?

Comment (demonwy) [reply shared/tasks/run-bats.sh]: @dpb587-pivotal  thanks for your advice. 
I think CPI cant not change disk size without the consent of the user.
@cdutra  any suggestion?

Comment (cdutra) [reply shared/tasks/run-bats.sh]: @dpb587-pivotal @demonwy If the IaaS doesn't support that specific size it should fail if specifying smaller than acceptable size. Also it should be documented in the cpi docs on bosh.io

Comment (xiaozhu36) [reply shared/tasks/run-bats.sh]: HI @cdutra I have added the note for disk size. Please refer to https://github.com/cloudfoundry/docs-bosh/pull/429/commits/66ee792fb549c54090a345a7a92c13188a5ec9c1 .

**Pull request closed without merging**